op-challenger/game/fault/agent.go

```diff
@@ -156,9 +163,13 @@ func (a *Agent) tryResolveClaims(ctx context.Context) error {
 	if len(claims) == 0 {
 		return errNoResolvableClaims
 	}
+	maxChessTime := a.gameDuration / 2
 
 	var resolvableClaims []uint64
 	for _, claim := range claims {
+		if claim.ChessTime(a.l1Clock.Now()) < maxChessTime {
```

Comment:
```suggestion
		if claim.ChessTime(a.l1Clock.Now()) <= maxChessTime {
```

Claims cannot be resolved if their chess time equals `maxChessTime`.
See FDG handling - https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol#L414

Reply:
True, but the next block will have a timestamp at least 1 greater and we're using the l1Clock which is set to the current head block.  That said, I'm not sure if `estimateGas` will use the current time or advance the clock so it's probably worth being conservative here - will change to `<=`.